frontend/src/logic/real-schedule-logic/children-info.logic.ts

```diff
@@ -0,0 +1,35 @@
+import { DataRow } from "./data-row";
+import { SectionLogic } from "./section-logic.model";
+
+export class ChildrenInfoLogic implements SectionLogic {
+  private childrenInfoAsDataRows: { [key: string]: DataRow } = {};
+
+  constructor(private childrenInfo: { [key: string]: number[] }) {
+    Object.keys(childrenInfo).forEach((key) => {
+      this.childrenInfoAsDataRows[key] = new DataRow(key, childrenInfo[key]);
+    });
+  }
+
+  public get data() {
+    return Object.values(this.childrenInfoAsDataRows);
+  }
+
+  public get sectionData() {
+    return this.data;
+  }
+
+  public get registeredChildrenNumber() {
+    // TODO refactor
+    return this.childrenInfoAsDataRows["liczba dzieci zarejestrowanych"]
```

Comment:
Refactor, ponieważ myślę, że było by fajnie dostawać dane o wszystkich dzieciach, a ta logika dostaje tylko dane które są w modelu wysyłanym na backend ( czyli tylko liczbę dzieci zarejestrowanych. ) Ale myślę, że rozwiążemy ten problem nie wcześniej niż kiedy dostaniemy harmonogramy

Reply:
Zgadzam się, tylko może w tym TODO jakoś byś to opisał, żebyśmy wiedzieli o co chodziło

Reply:
Słusznie, dodałem opis